account/lnurl.go

```diff
@@ -118,15 +118,21 @@ func (a *Service) HandleLNURL(rawString string) (result *data.LNUrlResponse, err
 			tag: "payRequest" // type of LNURL
 		}
 		*/
-		a.log.Info("lnurl.LNURLPayResponse1.")
+		a.log.Info("lnurl.LNURLPayParams.")
 		host := ""
 		if url, err := url.Parse(rawurl); err == nil {
 			host = url.Host
 		}
 		a.lnurlPayMetadata.encoded = params.EncodedMetadata
-		a.lnurlPayMetadata.data = params.Metadata
+		a.lnurlPayMetadata.data = [][]string{}
 		var metadata []*data.LNUrlPayMetadata
-		for _, e := range params.Metadata {
+
+		var adapter = [][]string{
```

Comment:
The go ln url  library has changed the Metadata from a `[][]srting` to a `Metadata` https://github.com/fiatjaf/go-lnurl/blob/master/pay.go#L72

As I'm not able to replicate how the proto buffers models are being generated on the project I did this adaptation with the current model, but I believe we should create a proper model, to do it I need a little help 😬

Reply:
two options here:
1. send me the messages.proto file that describes the new model and I will generate the code
2. Contat me offline and we'll try to fix it together. I am available